atm.js

```diff
@@ -1,21 +1,71 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+window.onload = function () {
+ $("div.account").addClass("zero");
+ };
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+var checkingBalance = parseFloat($("div#checking_balance").html().replace("$", ""));
+var savingsBalance = parseFloat($("div#savings_balance").html().replace("$", ""));
 
-}
+$("input[value='Deposit']").eq(0).on("click", function(){
+    var deposit = parseFloat($("input#checking_input").val());//Set their input equal to amount
+    checkingBalance += deposit;//Add amount to checkingBalance, i.e. checkingBalance += amount
+    $("div#checking_balance").html("$ " + checkingBalance.toFixed(2))//Update the "balance" div to reflect the new checkingBalance
+    if(checkingBalance > 0) {
+      $("div.account").eq(0).removeClass("zero");//Remove class zero
+    };
+})
 
-function checkingWithdrawal(amount){
+$("input[value='Withdraw']").eq(0).on("click", function(){
+  //When the user presses the withdrawal button in the first "account" div...
+    var withdrawal = parseFloat($("input#checking_input").val());//Set their input equal to amount
+    if((checkingBalance - withdrawal) < 0){
+      if(((checkingBalance + savingsBalance) - withdrawal) > 0){
+        savingsBalance -= (withdrawal - checkingBalance);
+        $("div#savings_balance").html("$ " + savingsBalance.toFixed(2));//Update the savings balance div
+        checkingBalance = 0;
+        $("div.account").eq(0).addClass("zero");
+      }
+      else {
+        alert("Insufficient funds");
+      }
+    }
+    else {
+      checkingBalance -= withdrawal;
+      if (checkingBalance == 0) {
+        $("div.account").eq(0).addClass("zero");
+      }
+    }
+  $("div#checking_balance").html("$ " + checkingBalance.toFixed(2));//Update the "balance" div to reflect the new checkingBalance
+})
```

Comment:
i noticed you used `parseFloat()` above and is subsequently why you have to use `.toFixed` here I think this way is fine. But for the scope of this problem it may be easier to user `parseInt()` I think either way works. In fact your way is better because it incorporates cents... now that i think about it.